### PR TITLE
LPS-188406 Use the edition Language instead of the user language and stop sending language to OpenAI

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPS-188406

cc: @p2kmgcl ❤️ 

This PR is only related to LPS-188406 but includes the LPS-182829 commits that are already merged in Brian's repo in order to prevent rebase conflicts.

### CODE REVIEW

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/ai-creator/ai-creator-openai-web/src/test/java/com/liferay/ai/creator/openai/web/internal/client/AICreatorOpenAIClientTest.java [hunk 1]
```diff
@@ -461,6 +461,9 @@ private void _testUnauthorizedResponseCode(
 		}
 
 		_assertOptions(apiKey, content, contentType, location);
+
+		_assertResponseJSONObject(responseJSONObject);
```
[the remark this answers is not in the document]
Comment (lfbesada) [reply modules/apps/ai-creator/ai-creator-openai-web/src/test/java/com/liferay/ai/creator/openai/web/internal/client/AICreatorOpenAIClientTest.java]: Thanks for catching it 😍 I have changed my commit to fit it!

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Review by ruben-pulido — APPROVED** [review]

**Comment by lfbesada** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**